resources/views/fashi/admin/category/index.blade.php

```diff
@@ -0,0 +1,168 @@
+@extends('admin_master')
+
+@section('content')
+<br>
+<div class="container">
+    <div class="row mb-2">
+        <div class="col-sm-6">
+            <h1>Category</h1>
+        </div>
+    </div>
+
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#exampleModalCreate">{{ trans('text.create_category') }}</button>
+    <br><br>
+
+    @if (session('message'))
+        <div class="alert alert-success" role="alert">
+            {{ session('message') }}
+        </div>
+    @endif
+
+    @error('name')
+        <div class="alert alert-danger" role="alert">
+            {{ $message }}
+        </div>
+    @enderror
+
+    <div class="modal fade" id="exampleModalCreate" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel">{{ trans('text.new_category') }}</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                      <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <form method="POST" action="{{ route('admin.categories.store') }}" enctype="multipart/form-data">
+                        @csrf
+                        @method('POST')
+                        <div class="form-group">
+                            <label for="name" class="col-form-label">{{ trans('text.name') }}:</label>
+                            <input type="text" class="form-control  @error('name') is-invalid @enderror" id="name" name="name" value="{{ old('name') }}" required="">
```

Comment:
những chỗ để cùng dòng input cũng thế nhé 
```suggestion
                            <input type="text" class="form-control  @error ('name') is-invalid @enderror" id="name" name="name" value="{{ old('name') }}" required="">
```
```suggestion
                            <input type="text" class="form-control  @error('name') is-invalid @enderror" id="name" name="name" value="{{ old('name') }}" required="">
```